### DISCUSSION TITLE
Invitation to Post-Quantum Cryptography Conference in Amsterdam

### DISCUSSION THREAD

**Discussion by linuxdevel** [pkic/community#30] [General]
Hi

I received an invitation to join the mentioned conference. In the invitation there is a link to the "conference page on our webpage". This link ends up in a 404 error. Can you please resend the invitation with correct link? I could'nt find any information about the new conference while browsing your site either.

**Comment by vanbroup** [comment]
The link is https://pkic.org/events/2023/pqc-conference-amsterdam-nl/

**Reply by linuxdevel** [reply]
And that link ends up in a 404 error.

**Reply by vanbroup** [reply]
ok, I purged the cache globally, it works from here, should be available globally, can you check again?

**Reply by linuxdevel** [reply]
Now it works, purging cache did the trick :) Thanks.
